cypress/integration/magic_8_ball_spec.js

```diff
@@ -0,0 +1,113 @@
+describe('Magic 8 ball user interaction', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://api.teleport.org/api/urban_areas/', {
+      statusCode: 200,
+      fixture: 'allCities.json'
+    });
+    cy.intercept(
+      'GET',
+      'https://api.teleport.org/api/urban_areas/slug:aarhus/scores/',
+      {
+        statusCode: 200,
+        fixture: 'aarhus_scores.json'
+      }
+    );
+    cy.intercept(
+      'GET',
+      'https://api.teleport.org/api/urban_areas/slug:aarhus/images/',
+      {
+        statusCode: 200,
+        fixture: 'aarhus_images.json'
+      }
+    );
+  });
+
+  it('after button click the 8 ball should shake', () => {
+    cy.visit('http://localhost:3000');
```

Comment:
Love how you checked for all of your status codes here. Since you're using the cy.visit and cy.get('button') with each test you may want to consider putting those in your beforeEach to shorten those it blocks!